packages/react-magma-dom/src/components/Typography/index.tsx

```diff
@@ -378,6 +378,8 @@ export const headingMediumStyles = props => css`
     @media (min-width: ${props.theme.breakpoints.small}px) {
       font-size: ${props.theme.typographyExpressiveVisualStyles.headingMedium
         .desktop.fontSize};
+      font-weight: ${props.theme.typographyExpressiveVisualStyles.headingSmall
```

Comment:
Should this be `props.theme.typographyExpressiveVisualStyles.headingMedium.fontWeight`?

Reply:
Actually desktop and mobile differ for Medium in regards to font-weight, desktop needs 600 where mobile needs 500 which is currently configured here.

Reply:
@orion-cengage is it absolutely necessary for desktop and mobile to have different font weights?

Reply:
I will verify with Marketing and report back.